openc3-cosmos-init/init.sh

```diff
@@ -1,6 +1,24 @@
 #!/bin/sh
 # set -x
 
+date
+# Run gem pristine on all gems
+# This ensures gems keep working on container upgrades
+# and if you change architectures
+previous=""
+for f in /gems/gems/* ; do
+  x=${f%.gem}
+  y=${x##*/}
+  z=${y%-*}
+
+  if [ "$previous" != "$z" ]
+  then
+    gem pristine $z
+  fi
+  previous=$z
+done;
+date
```

Comment:
Why do we need do to this? I didn't even know about gem pristine. I found [this article](https://www.justinweiss.com/articles/3-quick-gem-tricks/) that talks about simply doing `gem pristine --all --only-executables`. Not sure if that's a shortcut or if we need to do them all individually.

Reply:
This isn't required yet, but it will allow cosmos-project's to work across platforms, if we include the /gems into a local mount (generally).